pandas-stubs/core/arrays/boolean.pyi

```diff
@@ -1,24 +1,11 @@
 import numpy as np
 
-from pandas._typing import (
-    Scalar,
-    type_t,
-)
-
```

Comment:
shouldn't any changes to `core/arrays/boolean.pyi` not be in this PR?

Reply:
Removed these